### PR TITLE
Readme and License

### PR DESCRIPTION
**Pull request by garrying**
@dcwalk Carried over GPL-3.0 from tomesh repo. Do you have thoughts on the license for this?  Feel free to change if there is another you had in mind!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ournetworks.ca
```
Comment (dcwalk) [inline README.md:1]: Nice, a couple issues:
1. Could we add a one-sentence point to the planning repo, or link to the page or something for 2018?
1. We have to add a license & copyright line, needs to be assigned to individuals b/c we aren't an entity
1. I think best practice is we need to acknowledge the non-GPL and already licensed material (this includes normalize) and we should attribute to tomesh the 2017 material

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+[![Build Status](https://travis-ci.org/ournetworks/ournetworks.ca.svg?branch=master)](https://travis-ci.org/ournetworks/ournetworks.ca)
+
+This repository holds the source code and static assets of [ournetworks.ca](http://ournetworks.ca), built with [Jekyll](https://jekyllrb.com/).
+
+## Development
+1. Clone repository: `git clone https://github.com/ournetworks/ournetworks.ca.git && cd ournetworks.ca`
+2. Install Bundler gem: `gem install bundler`
+3. Install dependancies: `bundle install`
+4. Run locally: `bundle exec jekyll serve`
+5. Visit http://localhost:4000
```
Comment (dcwalk) [inline README.md]: My localhost is different b/c why not, so maybe we want to adjust this line?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# ournetworks.ca
+
+[![Build Status](https://travis-ci.org/ournetworks/ournetworks.ca.svg?branch=master)](https://travis-ci.org/ournetworks/ournetworks.ca)
+
+This repository holds the source code and static assets of [ournetworks.ca](http://ournetworks.ca), built with [Jekyll](https://jekyllrb.com/).
```
Comment (dcwalk) [inline README.md]: Should probably add Normalize here, and if the svgs were grabbed from anywhere?

File: README.md (hunk 1 above)
Comment (dcwalk) [reply README.md:1]: Also, RE: licensing, we need to add a blurb about site copy, I'd go the CC BY SA as we did that for the previous year (2017)

**Review by dcwalk — COMMENTED** [review]
See inline comments

File: README.md (hunk 1 above)
Comment (garrying) [reply README.md:1]: @dcwalk Made edits from the feedback. Re: issue 2, I'm a little unsure on how to best approach it. Maybe this is something you can append to this PR?

**Review by dcwalk — APPROVED** [review]
Hey @garrying sorry for letting this linger! I think it looks fine, I tried to mock up what I was getting at RE: 2, lemme know if you think it looks okay?

**Comment by garrying** [comment]
@dcwalk thanks for the follow up! looks good

**Pull request merged**